js/company.js

```diff
@@ -2,17 +2,22 @@ class Company{
     constructor(name) {
         this.name = name;
         this.count = 0;
+        this.pagesSeenOn = 0;
     };
 
     incrementCount(){
         this.count += 1;
     };
 
-    getCount(){
-        return this.count;
+    incrementPagesSeenOn(){
+        this.pagesSeenOn += 1;
     };
 
-    setCount(newCount){
-        this.count = newCount;
+    get(name){
+        return this[name];
+    };
```

Comment:
This is still super confusing at a glance. If you mean `get(property)` then it should say that, because there is a property called name in this class.